app/pages/exportImport/PerformImport.tsx

```diff
@@ -178,16 +267,30 @@ export default function PerformImport({ location }: Props) {
         importedData.accounts
             .filter((account: Account) => account.identityId === identity.id)
             .map((account: Account) => (
-                <p key={account.address}>{account.name}</p>
+                <p key={account.address}>
+                    {account.name}{' '}
+                    <span className="bodyLight">
+                        {messages[account.address] &&
+                            `(${messages[account.address]})`}
+                    </span>
```

Comment:
Maybe we should add a different color for the message as well? Also the `span` could be moved to avoid empty elements

```suggestion
                    {messages[account.address] &&
                        <span className="bodyLight textFaded">({messages[account.address]})</span>
                    }
```

Reply:
I agree, I've added this.